gapir/cc/android/asset_resource_cache.cpp

```diff
@@ -148,6 +151,9 @@ bool AssetResourceCache::loadCache(const Resource &resource, void *data) {
         // directly touch all pages of the destination memory to get the memory
         // tracker in a good state, and retry the read(). But try this only
         // once.
+        GAPID_WARNING(
```

Comment:
This will get very spammy, if this actually happens.
We should consider only having this output once, or only periodically.

Reply:
Removed.